front-end-notes/bootstrap.md

```diff
@@ -0,0 +1,28 @@
+#Bootstrap
+Bootstrap is a framework that allows developers to make front-end content more user friendly, quickly.  It is the most popular HTML, CSS, and JS framework for developing responsive, mobile first projects on the web.  It was developed by Mark Otto and Jacob Thornton at Twitter.  Bootstrap has designed templates for interface compontents such as nav bars, buttons, modals and other interface components.  
+
+Bootstrap includes a responsive, mobile first fluid grid system that appropriately scales up to 12 columns as the device or viewport size increases. It includes predefined classes for easy layout options for example btn for buttons.
+
+NSS Cohort 10 learned Bootstrap [v3.3.5](http://getbootstrap.com/) but [v4 alpha](http://blog.getbootstrap.com/2015/08/19/bootstrap-4-alpha/) is out with 4 beta being released soon.
+
+##Remember This
+- Grid system is divided into rows and each row can have up to 12 columns.
+
+- There can only be one container per page when using the grid system.
+
+- The Bootstrap class can be added directly to your html tags to provide the desired styling (it's that easy).
+
+- If using the javascript functionality with Bootstap jQuery must be  installed in the application.
+
+- Media queries can be used to customize styling according to device screan widths.
+
```

Comment:
Maybe include that funky shim stuff you need when using bootstrap with requireJS.